diracx-core/src/diracx/core/config/schema.py

```diff
@@ -159,6 +159,56 @@ class OperationsConfig(BaseModel):
     ResourceStatus: MutableMapping[str, Any] | None = None
 
 
+class DBConfig(BaseModel):
+    DBName: str
+    Host: str
+    Port: int
+
+
+class JobDBConfig(DBConfig):
+    MaxRescheduling: int = 3
```

Comment:
Move this to Services  `JobSchedulingConfig` further up

Reply:
And update the tests https://github.com/DIRACGrid/diracx/blob/main/diracx-cli/tests/legacy/cs_sync/integration_test.cfg nd the yaml.